### DISCUSSION TITLE
Does prometheus-bosh uses containers ?

### DISCUSSION THREAD

**Discussion by saketlj** [bosh-prometheus/prometheus-boshrelease#427] [General]
Hi,
Does prometheus bosh release use containerisation, if yes which container technology is used.
I have bosh deployment for prometheus and under that we have few vm's running firehose_exporte, cf_exporters,grafana, alertmanager etc. But need to understand if this are run within container.

Thanks.

**Comment by psycofdj** [comment]
Hello
There is no containerisation on prometheus vms. 
Very few jobs are wrapped by [bpm](https://github.com/cloudfoundry/bpm-release) which run process into namespaced cgroups. 
The only two jobs ran by bpm are `credhub_exporter` and `ingestor_exporter`.
